layouts/rss.xml

```diff
@@ -28,19 +31,18 @@
     {{ with .OutputFormats.Get "RSS" }}
     {{ printf "<atom:link href=%q rel=\"self\" type=%q />" .Permalink .MediaType | safeHTML }}
     {{ end }}
-    {{ range $pages }}
+    {{ range $pages }}{{ if ne .Params.rss_ignore true }}
     <item>
       <title>{{ .Title }}</title>
       <link>{{ .Permalink }}</link>
       <pubDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</pubDate>
-      {{ with .Site.Author.email }}<author>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</author>{{end}}
+      {{ with .Site.Author.email }}<author>{{.}}{{ with .Site.Author.name }} ({{.}}){{end}}</author>{{end}}
       <guid>{{ .Permalink }}</guid>
-      {{ if .Site.Params.rssFullContent }}
-        <description>{{ .Content | html }}</description>
-      {{ else }}
-        <description>{{ .Summary | html }}</description>
-      {{ end }}
+      <description>{{ with .Description | html }}{{ . }}{{ else }}{{ .Summary | html }}{{ end -}}</description>
+      {{- if not .Site.Params.RSSNoContent}}
```

Comment:
Could we make it backward-compatible and just use the original `.Site.Params.rssFullContent`?

Reply:
Sure, I'll make the change. Thank you!

Reply:
> Could we make it backward-compatible and just use the original `.Site.Params.rssFullContent`?

I changed it and commit again.